test/sharness/t0066-migration.sh

```diff
@@ -13,7 +13,7 @@ test_init_ipfs
 test_expect_success "setup mock migrations" '
   mkdir bin &&
   echo "#!/bin/bash" > bin/fs-repo-migrations &&
-  echo "echo 5" >> bin/fs-repo-migrations &&
+  echo "echo 10" >> bin/fs-repo-migrations &&
```

Comment:
Had to change this here to make the detection of the `fs-repo-migrations` to work since it seems now the code also checks the version the migration tool is able to handle.